scripts/release.js

```diff
@@ -1,211 +1,183 @@
-  process.exit(1);
-}
-let versionString = '';
-switch (userInput) {
-  case '1':
-  case '5':
-    version[2] += 1;
-    versionString = version.join('.');
-    break;
-  case '2':
-    version[1] += 1;
-    version[2] = 0;
-    versionString = version.join('.');
-    break;
-  case '3':
-    version[0] += 1;
-    version[1] = 0;
-    version[2] = 0;
-    versionString = version.join('.');
-    break;
-  case '4':
-    versionString = prompt(
-      util.format(
-        // Color prompt
-        '\x1b[34m%s\x1b[0m',
-        'Specify the version in Semantic Versioning format:'
-      )
-    )
-      .toString()
-      .trim();
-    break;
-  default:
-    process.exit(1);
-}
-const release_option = userInput;
-
-package_json['version'] = versionString;
-const versionFile = 'src/versioning/Versioning.ts';
-const changeLog = 'CHANGELOG.md';
-const tag = `amazon-chime-sdk-component-library-react@${versionString}`;
-logger.warn(`
-    Warning: you are bumping the version
-    `);
-logger.log(
-  `        From: ${util.format('\x1b[32m%s\x1b[0m', originalVersion)} `
-);
-logger.log(
-  `        
-            To:   ${util.format('\x1b[31m%s\x1b[0m', versionString)} `
-);
-
-shouldContinuePrompt();
-
-spawnOrFail('git', ['fetch origin']);
-if (release_option !== '5') {
-  logger.warn(`
-    Warning: are you sure to reset the HEAD to origin/main?
-    Current staged and local changes will be lost!\n`);
-  shouldContinuePrompt();
-  logger.log('Reseting HEAD to origin/main');
-  spawnOrFail('git', ['reset --hard origin/main']);
-}
-spawnOrFail('git', [' clean -ffxd .']);
-
-
-logger.log(`Updating ${versionFile} with ${versionString}`);
-
-// Update version number in Versions file.
-fs.writeFileSync(
-  versionFile,
-  fs
-    .readFileSync(versionFile)
-    .toString()
-    .replace(/return '[.0-9]+';/, `return '${versionString}';`)
-);
-
-logger.log(`Updating CHANGELOG.md with a new release entry - ${versionString}`);
-// Update CHANGELOG.md with release version
-fs.writeFileSync(
-  changeLog,
-  fs
-    .readFileSync(changeLog)
-    .toString()
-    .replace(
-      /\[Unreleased\]/,
-      `[${versionString}] - ${new Date().toISOString().slice(0, 10)}`
-    )
-);
-spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
-logger.log(`Updated package.json version to ${versionString}`);
-
-const updatedSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`]).trim();
-
-// Skip updating peer dependencies for hotfix
-if (release_option !== '5') {
+
+const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutput: true })).trim();
+
+const latestNPMJSSdkVersion = getLatestNPMJSSdkVersion(currentVersion);
+
+const deployDemo = (version) => {
+  logger.log('Deploying unique release candidate Meeting Demo URL...');
+  // Checkout the meeting demo
+  process.chdir(path.join(__dirname, '../..'));
+
+  let demoBranch = 'main';
+  if (!isCurrentMajorVersion()) {
```

Comment:
`isCurrentMajorVersion` is a bit confusing. Would suggest renaming to `isNextMajorVersion`? I checked the implementation then could understood it.

Reply:
Actually `isCurrentMajorVersion` is the correct name as we only have current and previous. Right now for example, current version is v3 (even though it is still in beta phase). Next major version is v4 :)

Reply:
Yeah I think whole wording of previous and current for me is confusing as actually for the builders the current stable will be our previous and next major version beta is the next major version. Since, we follow prev, current elsewhere this is fine.